include/crow/version.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
+
+constexpr char VERSION[] = "master";
```

Comment:
shouldn't this be similar to mime types when it comes to `CROW_MAIN`?

Reply:
`VERSION` is used in places without ifdef.

Reply:
Well what I meant is that it is a global variable (which makes it similar to mime types in that 2 different source files could end up defining it). Another thing I noticed is that it's outside the `crow` namespace, which might have it conflict with other global variables..

Reply:
Oh, ok ;) I've moved it to the namespace and add ifdef then ;)